src/components/shared/Slider/index.tsx

```diff
@@ -30,6 +30,19 @@ const ThumbPositioner = styled.View<ThumbPositionerType>`
   left: ${({ percent }): string => `${percent}%`};
 `;
 
+interface SliderStyle {
```

Comment:
I think the style props should be in `Props` without additional interfaces.
`LabelProps` as well.